src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx

```diff
@@ -0,0 +1,190 @@
+import React, { useState } from 'react';
+import {
+  getVideoKind,
+  isAtlasLocation,
+  getVAAppointmentLocationId,
+  isVideoHome,
+  isVideoGFE,
+  getATLASLocation,
+  isVideoAppointment,
+} from '../../../services/appointment';
+import { VIDEO_TYPES } from '../../../utils/constants';
+import VideoLink from './VideoLink';
+import AtlasLocation from './AtlasLocation';
+import VAFacilityLocation from '../../../components/VAFacilityLocation';
+import AlertBox from 'applications/gi/components/AlertBox';
+import { ALERT_TYPE } from '@department-of-veterans-affairs/component-library/AlertBox';
+import FacilityPhone from 'applications/vaos/components/FacilityPhone';
+import VideoVisitProvider from './VideoVisitProvider';
+import { VideoVisitInstructions } from './VideoInstructions';
+import AddToCalendar from 'applications/vaos/components/AddToCalendar';
+import moment from 'applications/vaos/lib/moment-tz';
+import { formatFacilityAddress } from 'applications/vaos/services/location';
+import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
+
+// Only use this when we need to pass data that comes back from one of our
+// services files to one of the older api functions
+function parseFakeFHIRId(id) {
+  return id ? id.replace('var', '') : id;
+}
+
+function getLocation(
+  isAtlas,
+  isVideo,
+  videoKind,
+  isCommunityCare,
+  facility,
+  appointment,
+) {
+  if (isAtlas) {
+    const atlasLocation = getATLASLocation(appointment);
+    if (atlasLocation?.address) {
+      return formatFacilityAddress(atlasLocation);
+    }
+  } else if (videoKind === VIDEO_TYPES.clinic) {
+    return facility ? formatFacilityAddress(facility) : null;
+  } else if (videoKind === VIDEO_TYPES.gfe) {
+    return 'Video conference';
+  } else if (isVideo) {
+    return 'Video conference';
+  } else if (isCommunityCare) {
```

Comment:
You could remove the conditionals in here that aren't video related, but it won't break anything.

Reply:
Good catch, fixed.